### PR TITLE
Chester's red pen edits

### PR DESCRIPTION
**Pull request by ismayc**
Includes refresh of `rds` folder. Also includes previous `rds` files in `rds_old` folder if anything doesn't look quite right.

### CODE REVIEW

**Comment by ismayc** [comment]
@rudeboybert This is ready for your review.

File: 10-inference-for-regression.Rmd [hunk 1]
```diff
@@ -57,33 +57,33 @@ library(patchwork)
...
 
 Recall using simple linear regression \index{regression!simple linear} we modeled the relationship between
 
-1. A numerical outcome variable $y$, the instructor's teaching score and
-1. A single numerical explanatory variable $x$, the instructor's "beauty" score.
+1. A numerical outcome variable $y$ (the instructor's teaching score) and
+1. A single numerical explanatory variable $x$ (the instructor's "beauty" score).
 
-We first created an `evals_ch6` data frame that selected a subset of variables from the `evals` data frame included in the `moderndive` package. This `evals_ch6` data frame contains only the variables of interest for our analysis, in particular the instructor's teaching `score` and the "beauty" rating `bty_avg`:
+We first created an `evals_ch5` data frame that selected a subset of variables from the `evals` data frame included in the `moderndive` package. This `evals_ch5` data frame contains only the variables of interest for our analysis, in particular the instructor's teaching `score` and the "beauty" rating `bty_avg`:
```
Comment (rudeboybert) [inline 10-inference-for-regression.Rmd]: good catch on `evals_ch5`

File: 00-preface.Rmd [hunk 2]
```diff
@@ -42,20 +42,20 @@ knitr::include_graphics("images/flowcharts/flowchart/flowchart.002.png")
...
 1. Statistical inference with `infer`. Once again using your newly acquired data science tools, you'll unpack statistical inference using the `infer` package. In particular you'll:
     + Ch.\@ref(sampling): Learn about the role that sampling variability plays in statistical inference and the role that sample size plays in sampling variability.
-    + Ch.\@ref(confidence-intervals): Construct confidence intervals.
-    + Ch.\@ref(hypothesis-testing): Conduct hypothesis tests.
-1. Data modeling with `moderndive` (revisited): Armed with your understanding of statistical inference, you'll revisit and review the models you'll construct in Ch.\@ref(regression) & Ch.\@ref(multiple-regression). In particular you'll:
+    + Ch.\@ref(confidence-intervals): Construct confidence intervals using bootstrapping.
+    + Ch.\@ref(hypothesis-testing): Conduct hypothesis tests using permutation.
+1. Data modeling with `moderndive` (revisited): Armed with your understanding of statistical inference, you'll revisit and review the models you'll construct in Ch.\@ref(regression) and Ch.\@ref(multiple-regression). In particular, you'll:
```
Comment (rudeboybert) [inline 00-preface.Rmd]: Made this match Ch11. Also testing out github.com GUI multi-line comment feature!

Comment (ismayc) [reply 00-preface.Rmd]: I like it!

**Pull request merged**